tests/data/jsons/file_types.json

```diff
@@ -6,14 +6,15 @@
         {
             "name": "FileTypeAssociation",
             "description": "Testing file type association",
-            "icon": null,
+	    "icon": null,
```

Comment:
I think the indentation got wonky here.

Reply:
Yes, it looks like something got replaced by a tab.